common/src/main/java/whocraft/tardis_refined/common/tardis/themes/console/sound/ConsoleSoundProfile.java

```diff
@@ -0,0 +1,14 @@
+package whocraft.tardis_refined.common.tardis.themes.console.sound;
+
+public abstract class ConsoleSoundProfile {
+
+    public ConsoleSound throttleEnable;
+    public ConsoleSound throttleDisable;
+    public ConsoleSound doorOpen;
+    public ConsoleSound doorClose;
+    public ConsoleSound doorLocked;
+    public ConsoleSound generic;
```

Comment:
Perhaps use getters and/or setters rather than expose the raw fields if these are meant to be pre-defined instances.